MatrixSDK/Crypto/Data/Store/MXRealmCryptoStore/MXRealmCryptoStore.m

```diff
@@ -146,6 +146,30 @@ + (NSString *)primaryKey
 @end
 RLM_ARRAY_TYPE(MXRealmOlmOutboundGroupSession)
 
+@interface MXRealmSharedOutboundSession : RLMObject
+
+@property NSString *uniqueKey;
+@property NSString *roomId;
+@property NSString *sessionId;
+@property MXRealmDeviceInfo *device;
+@property NSNumber<RLMInt> *messageIndex;
+
+@end
+
+@implementation MXRealmSharedOutboundSession
++ (NSString *)primaryKey
```

Comment:
Do we need it? 
We can let realm do its own optimisation.

Reply:
Indeed :)